README.md

```diff
@@ -144,4 +144,4 @@ I would like to thank Microverse for creating such wonderful opportunity for tho
 
 This project is [MIT]() licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="https://github.com/Esmatullah12/portfolio/blob/main/LICENSE.md">back to top</a>)</p>
```

Comment:
The link to your license page still leads to an error. This is due to the fact that you have not provided the reference of the license file to the link on line 145. 
Just as you provided a link for your linkedIn page 

`[LinkedIn](https://www.linkedin.com/in/esmatullah-qadeemi-b9951821b/)`, 

you should do similar. eg: `[MIT](./LICENSE`)  or `[MIT](./license)`